bcgov/PIMS
# PIMS-1373 ParcelBuildings Restructure Entities

## 🎯 Summary

  
[PIMS-1371](https://citz-imb.atlassian.net/browse/PIMS-1373?atlOrigin=eyJpIjoiZTY0OGU1Y2MyYjdmNDk5MWIzYTA5MjkwYmRkYTRkMzUiLCJwIjoiaiJ9)


## Changes
- Removed AccessRequest entity
- Changed UpdatedOn columns to be automatically updated
- Remove ParcelBuilding entity
- Move PID/PIN fields to Property entity so both parcels and buildings will have them
- Create a view (BuildingRelations) that can be used to reference property/building associations
- Add new migration files to accommodate changes

## Testing
1. Run migrations (can just start API)
2. Check database to see the new changes.

## Future Work
- Update ETL so that the buildings are populated with PID/PIN if they have associated parcels in old PIMS



## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
